flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/GenericHiveMetastoreCatalog.java

```diff
@@ -363,4 +332,23 @@ public CatalogColumnStatistics getPartitionColumnStatistics(ObjectPath tablePath
 		throw new UnsupportedOperationException();
 	}
 
+	// ------ utils ------
+
+	/**
+	 * Filter out Hive-created properties, and return Flink-created properties.
+	 */
+	private Map<String, String> retrieveFlinkProperties(Map<String, String> hiveTableParams) {
```

Comment:
static?

Reply:
What's the key point of making it static given no other class uses it?

Reply:
The difference is that a static method doesn't require an instance of the class in order to invoke it. For example, if another static method of this class needs to call this method, without this one defined as static, one has to create an instance of the class to invoke the method. The general principle is that if a method doesn't access any of the object variables/methods, make it static.

Reply:
make sense to me. Fixed